dev-tools/performance/filebeat/filebeat.yml

```diff
@@ -0,0 +1,34 @@
+name: performance-test
+
+filebeat.inputs:
+- type: filestream
+  id: my-filestream-id
+  enabled: true
+  paths:
+    - /tmp/flog.log
+
+setup.template:
+  name: "filebeat-performance-test-%{[agent.version]}"
```

Comment:
Should this template also be part of the PR?

Reply:
I just realised you only change the name, not load a different template. So removing my comment again.